src/classes/HiFiCommunicator.ts

```diff
@@ -257,6 +257,54 @@ export class HiFiCommunicator {
         });
     }
 
+    /**
+     * Adjusts the perceived volume of another user for this communicator's current connection only.
+     * This can be used to provide a more comfortable listening experience for the client. If you need to perform moderation actions which apply to all users, use the {@link https://docs.highfidelity.com/rest/latest/index.html|Administrative REST API}.
+     * 
+     * To use this command, the communicator must currently be connected to a space. You can connect to a space using {@link connectToHiFiAudioAPIServer}.
+     * 
+     * @param hashedVisitId  The hashed visit ID of the user whose volume will be adjusted.
+     * Use {@link addUserDataSubscription} and {@link HiFiCommunicator.onUsersDisconnected} to keep track of the hashed visit IDs of currently connected users.
```

Comment:
Point developers to the `ReceivedHiFiAudioAPIData` object as well, and specifically the fact that they can find the appropriate `hashedVisitId` for a given `providedUserID` (if they're specifying `provideUserIDs`) by examining the `ReceivedHiFiAudioAPIData` objects returned from the data subscription. 

Side note: I don't love that from the customer/developer side, but I'm struggling to think of a better way to do this. Ideally we would allow the developer to use `providedUserID` here instead of `hashedVisitId`, but since we don't require `providedUserID` (nor do we enforce uniqueness of it) I don't see an obvious way to get there. So, for now, I think using `hashedVisitId` and just explaining in the comments here how a developer could map from `providedUserID` to `hashedVisitId` is gonna be the best way to go.